pyproject.toml

```diff
@@ -0,0 +1,26 @@
+[tool.black]
+target_version = ['py36', 'py37', 'py38']
+include = '\.pyi?$'
+exclude = '''
```

Comment:
I've extended default exclude list for faster file discovery. We don't need to format cache dirs.

Reply:
Is it really needed to include `.pyi` stub files?